hazelcast/src/main/java/com/hazelcast/spi/impl/NodeEngineImpl.java

```diff
@@ -234,7 +235,7 @@ public Object toObject(Object object) {
 
     @Override
     public boolean isActive() {
-        return node.isActive();
+        return node.getState() == NodeState.ACTIVE;
```

Comment:
My intention is to make caller aware of the all node states by removing `node.isActive()`, it's not `active/passive` anymore. 

`NodeEngine.isActive()` is called by over SPI level, mostly at that level only important thing is if instance is `ACTIVE` or not. But `Node.getState()` is used on lower levels where difference between `SHUTTING_DOWN` and `SHUTDOWN` is important.